src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
Yeah @forki projects move too fast to handle ;)
I think dotnetcli is fine (the tool is called 'dotnet' and it's a command line interface). No idea what ms is doing...